data/mods/Fuji_Structures/npc/classes.json

```diff
@@ -0,0 +1,33 @@
+[
+    {
+        "type" : "npc_class",
+        "id" : "NC_BUNKER_MERCHANT",
+        "name" : "Merchant",
+		"common" : false,
+        "job_description" : "",
+        "bonus_str" : { "rng" : [ -1, 2 ] },
+        "bonus_dex" : { "rng" : [ -1, 1 ] },
+        "bonus_int" : { "rng" : [ -1, 4 ] },
+		"bonus_per" : { "rng" : [ -1, 2 ] },
+        "skills" : [
+            {
+                "skill" : "ALL", "level" : {
+                    "mul" : [
+                        { "one_in" : 3 },
+                        {
+                            "sum" : [
+                                { "dice" : [ 2, 2 ] },
+                                { "constant" : 0 }
+                            ]
+                        }
+                    ]
+                }
+            },
+            { "skill": "gun", "bonus" : { "rng" : [ 1, 5 ] } },
+            { "skill": "rifle", "bonus" : { "rng" : [ 1, 5 ] } },
+            { "skill": "pistol", "bonus" : { "rng" : [ 1, 4 ] } },
+			{ "skill": "smg", "bonus" : { "rng" : [ 1, 3 ] } },
```

Comment:
Linting your files will save space, make things look nicer and save time for people to move this to vanilla later on.